imsv-docs-astro/src/content/docs/guides/testing/payment-simulator.mdoc

```diff
@@ -10,12 +10,12 @@ a fake merchant.
 To begin using the simulator, one must first create an Immersve test card with
 the following steps:
 
-- Visit https://app.immersve.com/config and toggle `ENABLE_TEST_MODE` to on
-- Go to https://app.immersve.com and sign in with a new metamask account (don’t
+- Visit [https://app.immersve.com/config]() and toggle `ENABLE_TEST_MODE` to on
```

Comment:
When I click the link it doesn't follow the URL and stays on the current page. I think you might need to repeat the URL inside of round brackets

Reply:
Fixed by using bracket url syntax